### ISSUE TITLE
how to specify `ignore_na` and `adjust` in `DataArray.rolling_exp`?

### ISSUE THREAD

**Issue by abcde-1447** [pydata/xarray#6397]
### What is your issue?

Hi, I am trying to compute the moving average like the following in pandas using `DataArray.rolling_exp`:

```
df.ewm(alpha=0.9, ignore_na=True, adjust=True).mean()
```

does anyone know how to specify `ignore_na` and `adjust` in `DataArray.rolling_exp`?

Thanks!

**Comment by max-sixty** [comment]
Unfortunately those aren't in the interface at the moment. The default `adjust` is `True`. 

I hadn't found any cases for `ignore_na=True`, though maybe this is one? If the missing values are common across the dimensions that this is broadest over, dropping missing values and reindexing by the existing indexes has the same semantics.

Could you explain your use case?

**Comment by abcde-1447** [comment]
Thanks for the comment!

A typical use case for `ignore_na=True` is when we have a dataframe which has different nan locations for different columns, e.g. `df = pd.DataFrame({'a': pd.Series([0,np.nan, np.nan, 1,2]), 'b': pd.Series([np.nan, np.nan, 0, 1,2])})`.  In this case we could technically split the dataframe into several pd.Series and then apply `ewm` separately, but it would be more convenient if we can do this directly on the dataframe.

**Comment by max-sixty** [comment]
Yes that definitely makes sense. I was wondering what the underlying use case was, such that you want to ignore the missing values in your calc?

I think this is a reasonable feature request, but would have to come from a contribution. It wouldn't be the most complex contribution — it would involve adding the code from the pandas routine into the [numbagg code](https://github.com/numbagg/numbagg/blob/main/numbagg/moving.py#L8), and then updating the kwargs in xarray.

I'd be happy to do the release of numbagg once the code is in, since we don't do regular releases like we do in xarray.
